Fix group chat issues

jrainville opened the pull request:
This was a special ops task Iuri gave to me to test and fix any issue I found in the group chat.

- I found is that if someone leaves the group, they weren't removed from the user list. Now they are.
- The add new member to group didn't work
- Fix context menu
- Fix pinned messages popup from the group info popup

status-im-auto commented:
#### :heavy_check_mark: [status-desktop/branches/macos/PR-5521#1](https://ci.status.im/job/status-desktop/job/branches/job/macos/job/PR-5521/1/) :small_blue_diamond: ~9 min 53 sec :small_blue_diamond: c30c8626 :small_blue_diamond: [:package: macos package](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-Desktop-220412-170933-c30c86-pr5521.dmg)

jrainville commented:
I made this a draft, because I realised there were other issues not listed in the test list. I'll add more commits and let you guys know when you can review

jrainville commented:
alright, you guys can review and test

abhi-status commented:
Thanks Jo!

Could you please look into the group chat name change not being reflected in the main Chat view ?

![image](https://user-images.githubusercontent.com/95302135/163358600-0f04f33e-8521-42f5-ad7c-796ffa767da7.png)

jrainville commented:
@abhi-status I fixed it. I also found that changing the channel name didn't update it in another instance, so I fixed that too.

You can test again.

stefandunca commented on ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml:
```diff
@@ -170,40 +171,35 @@ StatusModal {
             currentValue: pinnedCount
-            onClicked: Global.openPopup(pinnedMessagesPopupComponent, {store: popup.store})
+            onClicked: {
+                popup.store.messageStore.messageModule = popup.chatContentModule.messagesModule
```
I see this as a requirement of the PinnedMessagePopup?  Is the update of module instance coupled with user actions?
